### DISCUSSION TITLE
Library Logo

### DISCUSSION THREAD

**Discussion by KingDarBoja** [nartc/angular-three#5] [Ideas]
I am too lazy to write something but pretty much Chau asked me to provide some logo design.

![logos_angular_threejs](https://user-images.githubusercontent.com/30560560/120252198-766a2100-c249-11eb-9272-9bcf508d7600.png)

**Comment by beeman** [comment]
I am too lazy to explain why, but there is only 1 true winner here: number 5.

<img width="162" alt="image" src="https://user-images.githubusercontent.com/36491/120252527-63a41c00-c24a-11eb-88d7-602d975edfad.png">

**Reply by nartc** [reply]
I am in the same boat.

**Reply by Jefiozie** [reply]
I think indeed number 5, but I like 3 also

**Reply by trungvose** [reply]
I am in for 1, 3 and 5

**Reply by adisreyaj** [reply]
+1 for number 5

**Reply by profanis** [reply]
I am between 1 and 5

**Reply by vltansky** [reply]
+1 for number 5.
Also 3 and 5 can change according dark/light theme.

**Reply by mawelCaballero** [reply]
I am in for number 1 but i am crazy with number 2.

**Reply by Oyemade** [reply]
Number 1

**Reply by mokipedia** [reply]
For uniformity: 1, but 5 because I like it.

**Reply by eskwisit** [reply]
1️⃣
